.ci/before_install.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+if [[ -n "${CI_TARGET}" ]]; then
+  exit
+fi
+
+if [[ "${TRAVIS_OS_NAME}" == osx ]]; then
+  brew update
+fi
+pip install --user --upgrade pip
```

Comment:
It doesn't work (see also https://github.com/travis-ci/travis-ci/issues/41949, but I'd just keep it in there, maybe it will be fixed at some point.